andela/project-arakari
# Chore/13842523/integrate coveralls code coverage service

#### What does this PR do?
Integrates Coveralls code coverage service
 #### Description of Task to be completed?
Add Coveralls, edit the travs.yml to make sure build data is sent to coveralls
 #### How should this be manually tested?
check the README.md, it has two badges, build and coverage both pointing to develop
 #### Any background context you want to provide?
 #### What are the relevant pivotal tracker stories?
story with id #135842523
 #### Screenshots (if appropriate)
 #### Questions:
